etebase-server.ini.example

```diff
@@ -18,3 +18,6 @@ allowed_host1 = example.com
 [database]
 engine = django.db.backends.sqlite3
 name = db.sqlite3
+
+[database_options]
```

Comment:
Should it maybe be `database-options`? I don't remember the convention in ini files, but this rings true.